docs/8_updates.md

```diff
@@ -156,15 +68,17 @@ Then rebuild the application, but do not flash the binary to your development bo
 
 ### Updating the device
 
-Now we can push this new application to your development through Device Management. The manifest tool can both sign the update - using the private key generated earlier - and upload it to Device Management in a single command.
+An application that is managed by Device Management consists of two parts. Your application and a [bootloader](https://github.com/armmbed/mbed-bootloader). The bootloader verifies the firmware on the device and can swap firmware for other firmware. These two applications are automatically combined into a single binary when building. However, for firmware updates we want the binary without the bootloader. It's located in your build folder as `connected-lights-cloud_application.bin`.
+
+We can push this new application to your development through Device Management. The manifest tool can both sign the update - using the private key generated earlier - and upload it to Device Management in a single command.
```

Comment:
Two questions related to this line: 
1. Should this be "your development board" instead of "your developtment"?
1. Is the mention of manifest tool correct? The code that follows replaces the `manifest-tool` command with `mbed device-management` command.

Reply:
Thanks, fixed both.